src/ui/docengine.cpp

```diff
@@ -130,8 +168,10 @@ void DocEngine::loadDocuments(const DocEngine::DocumentLoader& docLoader)
         QString localFileName = url.toLocalFile();
         QFileInfo fi(localFileName);
 
-        QPair<int, int> openPos = findOpenEditorByUrl(url);
-        if(!reload && openPos.first > -1) {
+        const QPair<int, int> openPos = findOpenEditorByUrl(url);
+        const bool isReloading = openPos.first > -1; //'true' when we're reloading a tab
```

Comment:
Just to use a more meaningful naming, this could be "isAlreadyOpen" instead of "isReloading"